python/ray/_private/runtime_env/utils.py

```diff
@@ -280,3 +287,109 @@ def from_dict(
         _build_proto_container_runtime_env(runtime_env_dict, proto_runtime_env)
         _build_proto_plugin_runtime_env(runtime_env_dict, proto_runtime_env)
         return cls(proto_runtime_env=proto_runtime_env)
+
+
+class SubprocessCalledProcessError(subprocess.CalledProcessError):
+    """The subprocess.CalledProcessError with stripped stdout."""
+
+    LAST_N_LINES = 10
```

Comment:
```suggestion
    LAST_N_LINES = 50
```
Can we increase it to 50 just in case?  Sometimes 10 is not enough: https://github.com/ray-project/ray/pull/22198#discussion_r802323857

Reply:
Or maybe it makes sense to leave this default as 10, and pass 50 in when calling from runtime_env pip.

Reply:
The pip install output string is usually very long, but the error string is short at the end. I am OK to change the default to 50.

Reply:
Fixed.